### ISSUE TITLE
NTR: Prolidase activity low in erythrocytes, leukocytes, or skin fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8221]
New term request Prolidase activity low in erythrocytes, leukocytes, or skin fibroblasts (elthat_111113013505): RBC, WBC, red and white blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000828
- label: Reduced tissue peptidase D activity
- parents: [Abnormal tissue enzyme concentration or activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:3139929|PMID:12384772
- def: Concentration or activity of Peptidase D (EC 3.4.13.9; also known as prolidase, imidodipeptidase, proline dipeptidase, and aminoacyl-L-proline hydrolase) below the lower limit of normal. The activity of this enzyme can be measured in erythrocytes, leukocytes, or fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8221
- synonyms: Reduced tissue prolidase activity(NONE)
